ckanext/imageview/plugin.py

```diff
@@ -17,6 +17,8 @@ class ImageView(p.SingletonPlugin):
 
     def update_config(self, config):
         p.toolkit.add_template_directory(config, 'theme/templates')
+        image_formats = config.get('ckan.preview.image_formats', '').split()
+        self.formats = image_formats or DEFAULT_IMAGE_FORMATS
```

Comment:
Just saw this and deleted my last comment, but I think it's better to use the DEFAULT_IMAGE_FORMATS in the get() so that explicitly setting no image formats in the ini actually works (it's what the user requested, even if it's strange)